lib/WGmeta/Cli/Commands/Set.pm

```diff
@@ -0,0 +1,117 @@
+package WGmeta::Cli::Commands::Set;
+use strict;
+use warnings FATAL => 'all';
+use experimental 'signatures';
+
+require WGmeta::Cli::Commands::Command;
+use WGmeta::Wireguard::Wrapper::Config;
+our @ISA = qw(WGmeta::Cli::Commands::Command);
+
+use constant WIREGUARD_HOME => '/home/tobias/Documents/wg-meta/t/Data/';
+use constant TRUE => 1;
+use constant FALSE => 0;
+
+sub new($class, @input_arguments) {
+    my $self = $class->SUPER::new(@input_arguments);
+
+    bless $self, $class;
+
+    return $self;
+}
```

Comment:
I think this does not actually do anything ... since it is no different from the SUPER implementation, so the whole routine can go ...

```suggestion
```